npm_and_yarn/lib/dependabot/npm_and_yarn/update_checker/version_resolver.rb

```diff
@@ -332,22 +332,22 @@ def fetch_peer_dependency_errors(version:)
           []
         end
 
-        def handle_peer_dependency_errors(error)
+        def handle_peer_dependency_errors(message)
```

Comment:
nice cleanup. This one could perhaps be extracted to its own PR to keep the `pnpm` PR smaller?

Reply:
Sure.

Reply:
Done at #7178!